src/applications/disability-benefits/526EZ/tests/actions/actions.unit.spec.js

```diff
@@ -0,0 +1,275 @@
+import { expect } from 'chai';
+import sinon from 'sinon';
+
+import { mockFetch, mockApiRequest, resetFetch } from '../../../../../platform/testing/unit/helpers.js';
+
+import {
+  getLatestTimestamp,
+  isNotEmptyObject,
+  getITFsByStatus,
+  PRESTART_STATUS_SET,
+  PRESTART_DATA_SET,
+  PRESTART_STATE_RESET,
+  PRESTART_DISPLAY_RESET,
+  PRESTART_STATUSES,
+  PRESTART_VERIFICATION_TYPES,
+  ITF_STATUSES,
+  setPrestartStatus,
+  setPrestartData,
+  resetPrestartState,
+  resetPrestartDisplay,
+  handleCheckSuccess,
+  handleCheckFailure,
+  handleSubmitSuccess,
+  handleSubmitFailure,
+  checkITFRequest,
+  submitITFRequest,
+  verifyIntentToFile
+} from '../../actions';
+
+import existingData from '../itfData';
+
+const noData = {};
```

Comment:
Same as an above comment; since this represents what's being passed to the `handleCheckSuccess` function, `const noData = undefined;` is more accurate. If `get()` is used, it'll work either way, but may be a bit misleading if we test `data` as an empty object when it never will be, y'know?